cmd/root.go

```diff
@@ -307,11 +308,15 @@ func pullAndRebase(container string) (imgid string, changed bool) {
 		imgid = container
 	}
 
-	// Clean up a previous container
-	podmanRemove(types.PivotName)
```

Comment:
We might as well keep this line to ensure we GC containers with the previous naming.

Reply:
Done! (Though note the original patch didn't do this).